src/peakrdl_sv/__peakrdl__.py

```diff
@@ -5,39 +5,38 @@
 from peakrdl.plugins.exporter import ExporterSubcommandPlugin
 from systemrdl.node import AddrmapNode
 
-from .exporter import VerilogExporter
+from .exporter import PythonExporterBase
+from .exporter import VerilogExporterBase
 
 
-def validate_reset_polarity(string):
-    if string not in ("high", "low"):
-        raise argparse.ArgumentError
-    return string
-
-
-def validate_reset_type(string):
-    if string not in ("async", "sync"):
-        raise argparse.ArgumentError
-    return string
-
-
-class Exporter(ExporterSubcommandPlugin):
+class VerilogExporter(ExporterSubcommandPlugin):
     short_desc = "A SystemVerilog SystemRDL exporter"
 
     def add_exporter_arguments(self, arg_group: argparse.ArgumentParser) -> None:
         arg_group.add_argument(
             "--reset-polarity",
             default="high",
-            type=validate_reset_polarity,
+            type=str,
+            choices=["high", "low"],
```

Comment:
Unrelated to the PR but noticed that this could be simplified, unless the original approach was deliberate.

Reply:
That looks fine so you can remove the `validate*` method.

Reply:
Done, that change was also included in this PR.